File: Source File/C++/ForLoop_3.cpp
#include <iostream>
using namespace std;

int main(){
    int num;

    cout << "Enter any number (1-9) : " ;
    cin >> num;

    cout << "The table of " << num << " is " << endl;
    for(int i = 1; i <= 10; i++){
        cout << num << " x " << i << " = " << num * i << endl;
    }

    return 0;
}

File: Source File/Java/ForLoop_3.java
import java.util.Scanner;

public class ForLoop3{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);

        int i;
        System.out.print("Enter the number you want to print table of:");
        int num=sc.nextInt();

        for(i=1; i<=10; i++){
            System.out.println(num +" x " +i +" = " +num*i);
        }

    }
}